src/components/Sepa/components/IbanInput/IbanInput.js

```diff
@@ -13,15 +13,30 @@ class IbanInput extends Component {
         this.state = {
             status: 'ready',
             data: {
-                'sepa.ownerName': '',
-                'sepa.ibanNumber': ''
+                'sepa.ownerName': props.ownerName ? props.ownerName : '',
+                'sepa.ibanNumber': props.ibanNumber ? props.ibanNumber : ''
```

Comment:
Hey @emidev98, glad to see you contributing again!

A couple of comments here:
This `props.ibanNumber ? props.ibanNumber : ''` can be simplified like this `props.ibanNumber || ''`. We could as well use optional chaining like this `'sepa.ownerName': props?.ownerName`.

But also, to align with other components, I'd wrap both `ownerName` and `ibanNumber` in a `data` object.
```js
data: {
    'sepa.ownerName': props?.data?.ownerName,
    'sepa.ibanNumber': props?.data?.ibanNumber
}
```
We could as well include the properties only when necessary:
```js
data: {
    ...(props?.data?.ownerName && { 'sepa.ownerName': props.data.ownerName }),
    ...(props?.data?.ibanNumber && { 'sepa.ibanNumber': props.data.ownerName })
}
```
Keep in mind that tests will need to be updated as well.

Reply:
Make sense fro code standards inside the library.
Already did the change and wrote the tests